public/examples/js/offscreen-canvas/basic.js

```diff
@@ -0,0 +1,45 @@
+// This example is the based on demos-basic/container, but using OffscreenCanvas.
+
+const width = 800;
+const height = 600;
+const resolution = window.devicePixelRatio;
+const canvas = document.createElement('canvas');
+canvas.style.width = `${width}px`;
+canvas.style.height = `${height}px`;
+document.body.appendChild(canvas);
+const view = canvas.transferControlToOffscreen();
```

Comment:
Safari doesn't handle this, it would be good to gracefully handle this by showing an error message instead.

Reply:
Maybe we can add something like `"checks": ["OffscreenCanvas"]` in `manifest.json` and do the check in `main.js`, so that we can make the example codes simple?

Reply:
Yeah, I'm cool with doing checks in main

Reply:
Done!

![image](https://user-images.githubusercontent.com/8724868/200654245-2b9a8383-a197-4b9d-bc6b-0494a66a9d65.png)